### DISCUSSION TITLE
Google PlayStore version is now available !

### DISCUSSION THREAD

**Discussion by Nain57** [Nain57/Smart-AutoClicker#267] [Announcements]
After a long battle against Google security policies, I'm happy to announce you that Smart AutoClicker is now available on the PlayStore !
All futures releases will now be published on Github, F-Droid and Google PlayStore simultaneously.

Here is the link: https://play.google.com/store/apps/details?id=com.buzbuz.smartautoclicker

**Comment by Recress** [comment]
Congrats man!

**Comment by DrankRock** [comment]
I discovered it yesterday, after failing to make my own auto clicker (seriously I found no clear uide on how to simulate event for release (and not test) purpose). First time in a while I buy an app, this is awesome, thanks a lot.

**Comment by shmaungms** [comment]
Congratulations on your play store release.  I've left a 5 star review for you.
